src/Ombi.Api 2.Service/AppVeyorApi.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
```

Comment:
Any reason why it seems you can copied the folder?

Reply:
Seems like a bad manipulation on my side... Deleted the whole folder